content/en/docs/tasks/administer-cluster/coredns.md

```diff
@@ -19,11 +19,14 @@ This page describes the CoreDNS upgrade process and how to install CoreDNS inste
 
 ## About CoreDNS
 
-[CoreDNS](https://coredns.io) is a flexible, extensible DNS server that can serve as the Kubernetes cluster DNS.
-Like Kubernetes, the CoreDNS project is hosted by the {{< glossary_tooltip text="CNCF" term_id="cncf" >}}.
+[CoreDNS](https://coredns.io) is a flexible, extensible DNS server
+that can serve as the Kubernetes cluster DNS.
+Like Kubernetes, the CoreDNS project is hosted by the
+{{< glossary_tooltip text="CNCF" term_id="cncf" >}}.
 
-You can use CoreDNS instead of kube-dns in your cluster by replacing kube-dns in an existing
-deployment, or by using tools like kubeadm that will deploy and upgrade the cluster for you.
+You can use CoreDNS instead of kube-dns in your cluster by replacing
+kube-dns in an existing deployment, or by using tools like kubeadm
+that will deploy and upgrade the cluster for you.
```

Comment:
This might actually be outdated; is `kube-dns` still supported? (I think it might be at least deprecated).

Reply:
kube-dns is deprecated and removed from kubeadm
https://github.com/kubernetes/kubeadm/issues/1943

Reply:
If you don't use `kubeadm`, I'm assuming nothing stops you using KubeDNS (but hardly anyone does). Is that right?

Reply:
Correct. `kube-dns` is still a valid option that implements the DNS schema. It's just not the recommended one, nor the one installed by `kubeadm`.

Reply:
It is NOT deprecated as an implementation itself (i.e., https://github.com/kubernetes/dns is active and supported).